packages/editor/package.json

```diff
@@ -28,6 +28,7 @@
     "@codemirror/lang-markdown": "^6.2.0",
     "@codemirror/language": "^6.8.0",
     "@codemirror/language-data": "^6.3.1",
+    "@codemirror/merge": "6.0.0",
```

Comment:
最新版を使おうとすると依存関係の問題でエラーが起こるのでバージョンを固定しています

```
Unhandled Runtime Error
Error: Unrecognized extension value in extension set ([object Object]). This sometimes happens because multiple instances of @codemirror/state are loaded, breaking instanceof checks.
```

Reply:
修正しました